includes/Framework/Utilities/BackgroundJobHandler.php

```diff
@@ -163,14 +163,19 @@ protected function is_queue_empty() {
 		$queued     = '%"status":"queued"%';
 		$processing = '%"status":"processing"%';
 
-		$count = $wpdb->get_var( $wpdb->prepare( "
-			SELECT COUNT(*)
-			FROM {$wpdb->options}
-			WHERE option_name LIKE %s
-			AND ( option_value LIKE %s OR option_value LIKE %s )
-		", $key, $queued, $processing ) );
+		$count = $wpdb->get_var(
+			$wpdb->prepare(
+				"SELECT COUNT(*)
+				FROM {$wpdb->options}
+				WHERE option_name LIKE %s
+				AND ( option_value LIKE %s OR option_value LIKE %s )",
+				$key,
+				$queued,
+				$processing
+			)
+		);
 
-		return ( $count > 0 ) ? false : true;
+		return ! ( $count > 0 );
```

Comment:
why not just use `$count == 0`?

It fits well with the name of the method. Just a nitpick - it can be ignored.

Reply:
I did `intval( $count ) === 0`, maybe redundant, but we get a string inside the `count` and to pass code styling check we need `===`